diff --git a/dist/index.cjs b/dist/index.cjs
index 6f434ee828045269e365e493e96227f4f28c4a91..c3e928e47ee037e8e188e3276e28db99a17c6316 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -1,5 +1,10 @@
 'use strict';
 
+const {
+  name: packageName,
+  version: packageVersion
+} = require("../package.json");
+
 const parseForESLint = (code) => ({
   ast: {
     type: "Program",
@@ -16,4 +21,10 @@ const parseForESLint = (code) => ({
   }
 });
 
+const meta = {
+  name: packageName,
+  version: packageVersion,
+}
+
 exports.parseForESLint = parseForESLint;
+exports.meta = meta;
diff --git a/dist/index.mjs b/dist/index.mjs
index bdd48faa1bb1716f3037dfcf7a263c90022f40d0..aa7d894b2b760b9eed7afc3df32db87d5f6e677c 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1,3 +1,8 @@
+import {
+  name as packageName,
+  version as packageVersion,
+} from "../package.json";
+
 const parseForESLint = (code) => ({
   ast: {
     type: "Program",
@@ -14,4 +19,9 @@ const parseForESLint = (code) => ({
   }
 });
 
-export { parseForESLint };
+const meta = {
+  name: packageName,
+  version: packageVersion,
+}
+
+export { parseForESLint, meta };
